Packs/MicrosoftGraphGroups/Integrations/MicrosoftGraphGroups/MicrosoftGraphGroups.py

```diff
@@ -569,17 +407,14 @@ def main():
     """
     PARSE AND VALIDATE INTEGRATION PARAMS
     """
-    base_url = demisto.params().get('url').rstrip('/') + '/v1.0/'
-    tenant = demisto.params().get('tenant_id')
-    auth_and_token_url = demisto.params().get('auth_id').split('@')
-    auth_id = auth_and_token_url[0]
-    enc_key = demisto.params().get('enc_key')
-    verify = not demisto.params().get('insecure', False)
-    proxy = demisto.params().get('proxy') == 'true'
-    if len(auth_and_token_url) != 2:
-        token_retrieval_url = 'https://oproxy.demisto.ninja/obtain-token'  # guardrails-disable-line
-    else:
-        token_retrieval_url = auth_and_token_url[1]
+    params: dict = demisto.params()
+    base_url = params.get('url', '').rstrip('/') + '/v1.0/'
```

Comment:
i think this is handled in the BaseClient, please check and change accordingly

Reply:
Checked, it's not being handled within BaseClient. I'll keep it as is.